src/org/zaproxy/zap/spider/parser/SpiderHtmlFormParser.java

```diff
@@ -225,14 +214,15 @@ private void processGetForm(HttpMessage message, int depth, String action, Strin
 	 * @param form the form
 	 * @return the list
 	 */
-	private FormData prepareFormDataSet(FormFields form) {
+
+	public FormData prepareFormDataSet(FormFields form) {
 		List<HtmlParameter> formDataSet = new LinkedList<>();
 		List<HtmlParameter> submitFields = new ArrayList<>();
 
 		// Process each form field
 		Iterator<FormField> it = form.iterator();
 		while (it.hasNext()) {
-			FormField field = it.next();
+			  FormField field = it.next();
```

Comment:
Formatting changes should occur in a separate PR:
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting